### ISSUE TITLE
Batch Render or Loop Render

### ISSUE THREAD

**Issue by enzyme69** [benrugg/AI-Render#17]
### Describe the feature you'd like to see:

Since unlimited "local render" is working, maybe we need also the ability to batch render static or animated frames -- with different random seed or same seed. I would like to render over night usually, since my local render is kind of slow 4-10 minutes per image, I can let it render for 6-8 hours while asleep or when computer idle.

Another request is if we could "loop" render X times the image from last render or image last AI render, sometimes it could produce interesting details.

Thanks!

**Comment by benrugg** [comment]
Yeah, this is a great idea, and it's in my plans. I definitely want to make this happen.

**Comment by Ainaemaet** [comment]
Was just about to write up a feature request for this. Very happy to see it being worked on. We're blessed to have devs with a positive work-ethic.

Thank you @benrugg !!

**Comment by benrugg** [comment]
Batch processing is now possible with the animation support in the latest release! You can animate Stable Diffusion settings (including prompt text!) and run as many variations as you want. 

I will leave this open, though, because the one thing that isn't available yet is "loop rendering". ie. the ability to use the last AI image as the start for the next one.

**Comment by benrugg** [comment]
See the [Animation Instructions](https://github.com/benrugg/AI-Render/wiki/Animation) page for more info. (And I'll be posting more instructions there soon)

**Comment by TutajITeraz** [comment]
Hi, I'm trying to achieve something similar, but it seems to me that instead of adding one option, it's best to make Compositing Nodetree work properly. Currently, for some reason, anything after AIRender and before Composite node - is ignored and not saved, even if save before and after is checked. I tried saving it manually because the preview in "Render Result" appears correct.

**Comment by benrugg** [comment]
@TutajITeraz ah, that's a really good point. It's working that way by design, but tbh I probably didn't think that through well enough. It certainly seems like it should save the final composite, not just what was returned by Stable Diffusion. I'll add this to the list for the future.
